src/client-apps/nuxeo-drive.md

```diff
@@ -779,6 +779,10 @@ Direct Edit enables you to edit any of your document's content from their Summar
 
 The icon looks like ![]({{file name='drive-webui-direct-edit.png' page='nuxeo-drive'}} ?w=20) on Web-UI and ![]({{file name='drive-jsf-direct-edit.png' page='nuxeo-drive'}}) on JSF-UI.
 
+{{#> callout type='note' }}
```

Comment:
I have not the rendered page, but is not it a duplicate of the above line?

Reply:
The idea is to give a heads-up to the fact the icon changed in WebUI 10.10 version. Do you have any suggestion to make it more explicit?

Reply:
Nope, it is OK for me, I just wanted to be sure you saw it :)
I am +1 for the change.